src/handlers/messages-get.ts

```diff
@@ -2,7 +2,7 @@ import type { DataStore } from '../types/data-store.js';
 import type { DidResolver } from '../did/did-resolver.js';
 import type { MessageStore } from '../types/message-store.js';
 import type { MethodHandler } from '../types/method-handler.js';
-import type { RecordsWriteMessageWithOptionalEncodedData } from '../types/records-types.js';
```

Comment:
Even if only for renaming `RecordsWriteMessageWithOptionalEncodedData`, I am extremely supportive of this PR 😆